todoman/cli.py

```diff
@@ -80,6 +80,14 @@ def _validate_start_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_category(ctx, param, val):
+    ctx = ctx.find_object(AppContext)
+    try:
+        return ctx.formatter.parse_category(val)
+    except ValueError as e:
```

Comment:
For consistency I'd vote for leaving it in though. When editing `parse_category`, I would expect that `ValueError` is handled like it always is.